setup.py

```diff
@@ -40,7 +40,8 @@
 
 setup(
     name="cadquery",
-    use_scm_version=True,
+    # use_scm_version=True,
+    version="2.2.0",
```

Comment:
How about a release candidate or beta release first?  It could validate that all of the packages are built, uploaded, tagged, etc. correctly.  Or are you confident without doing this?

Reply:
We've already had multiple beta releases for 2.2.0, at least on [PyPI](https://pypi.org/project/cadquery/#history). The OCP 7.7.0a0 wheels have been tested to make sure all tests pass with them installed, and the conda packages are used to build the wheels (except for our custom VTK wheels on Linux now). So I feel pretty good about the release. I'm never 100% sure that things won't break because I've been bitten by over-confidence too many times, but I think we're well positioned for this release.

I'm confident enough that I think we go ahead with this one, and I'm not opposed to doing a 2.2.1 release if we find we need one. We could also talk about starting a December-January and May-June release-candidate phase for 2.3.0. That could go in the [Release Schedule](https://github.com/CadQuery/cadquery/discussions/1115) discussion.